### PR TITLE
TorqueObject action added

### PR DESCRIPTION
**Pull request by jaudick**
[Unity](https://github.com/NextCenturyCorporation/ai2thor/pull/191)

I updated the pickup object code because trying to pickup out of range objects would stop their torque instantly. Also test 45 was completely broken. The agent was clipping into the couch when picking up an object already in its hand. Fixing the pickup object fixed code that test.

### CODE REVIEW

**Review by deanwetherby — APPROVED** [review]

File: machine_common_sense/parameter.py [hunk 1]
```diff
@@ -39,6 +39,8 @@ class Parameter:
 
     MAX_FORCE = 1.0
     MIN_FORCE = 0.0
+    MAN_FORCE_TORQUE = 1.0
```
Comment (jaudick) [inline machine_common_sense/parameter.py:42]: Fixing this with an updated commit

**Comment by jaudick** [comment]
@ThomasSchellenbergNextCentury I merged in your push pull force changes here, let me know if they are not merged correctly with the addition of torque

File: integration_tests/data/095.torque_object.oracle.outputs.json [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+    {
+        "camera_height": 0.762,
+        "head_tilt": 30,
+        "objects": [
+            {
+                "id": "turtle_on_wheels",
+                "mass": 1.5,
+                "position_x": 0.0,
+                "position_z": 1.011,
+                "rotation_x": 360,
```
Comment (ThomasSchellenbergNextCentury) [inline integration_tests/data/095.torque_object.oracle.outputs.json:200]: Could we please check the `rotation_y` of the object on each steps as well? Seems like an important thing to verify for torquing.

Comment (jaudick) [reply integration_tests/data/095.torque_object.oracle.outputs.json:143]: Change implemented

**Review by ThomasSchellenbergNextCentury — APPROVED** [review]
Tests all run successfully. Great work!

**Pull request merged**